internal/cmd/clone/clone.go

```diff
@@ -19,14 +20,21 @@ func (o *options) AddFlags(cmd *cobra.Command) {
 		"",
 		"specify branch to check out",
 	)
+	cmd.Flags().StringSliceVarP(
+		&o.expectedRootKeys,
+		"branch",
+		"b",
+		[]string{},
```

Comment:
We ought to think about what the format for keys is. I want us to be sure key IDs is enough. Also, users would have to _know_ the key IDs when it might actually be easier to download and point to pubkey files directly.

Reply:
I have changed the input from the KeyId's themselves to the key's paths

Reply:
@wlynch do you have thoughts on how to specify keys here? Should it match other flags we specify keys with? i.e., path on disk / gpg:<> / fulcio:<>.